certbot/tests/plugins/common_test.py

```diff
@@ -177,7 +177,7 @@ def test_current_file_hash_in_all_hashes(self):
 
 
 class AddrTest(unittest.TestCase):
-    """Tests for certbot._internal.client.plugins.common.Addr."""
+    """Tests for certbot._internal.plugins.common.Addr."""
```

Comment:
I think this is located in `certbot.plugins.common` now!

Reply:
Ah you're right! Nice catch.